torch/quantization/fx/backend_config_dict/fbgemm.py

```diff
@@ -0,0 +1,12 @@
+import torch
+from ..quantization_patterns import *
+from ..pattern_utils import get_default_quant_patterns
+
+def get_fbgemm_backend_config_dict():
+    """ Get the backend config dictionary for fbgemm backend
+    NOTE: Current api is not final, it's just to unblock experimentation for new backends
+    """
+    # TODO: add output_activation_post_process_map
+    return {
+        "quant_patterns": get_default_quant_patterns()
```

Comment:
might be good to clarify further that this API will definitely change, and it's 100% unsafe for customers to depend on this in any way until further notice.